docs-src/releases.md

```diff
@@ -0,0 +1,48 @@
+<!--
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Releases
+
+## Official Custom Resource Definitions
+
+The custom resources defined as part of Service APIs will be considered official
+custom resource definitions (CRDs). That means that they will go through a KEP
+review process and be part of a Kubernetes project. 
+
+This is a relatively new concept, and there is only one previous example of this
```

Comment:
Let's move this to a "discussion" section below and put a straightforward description of the release policy/process up front.

Reply:
With all the changes I made to the structure here, I ended up using a "other official custom resources" section at the bottom to talk about VolumeSnapshots. Hopefully that makes sense here.